crates/hyperswitch_connectors/src/connectors/nomupay/transformers.rs

```diff
@@ -142,87 +381,254 @@ impl<F, T> TryFrom<ResponseRouterData<F, NomupayPaymentsResponse, T, PaymentsRes
+                        country_code: country_iso2_code,
+                        currency_code: item.request.destination_currency,
+                        transfer_method_type: "BANK_ACCOUNT".to_string(),
+                        display_name: item.get_billing_full_name()?,
+                        bank_account,
+                        profile,
+                    })
+                }
+                _ => Err(errors::ConnectorError::NotSupported {
+                    message: "SEPA payouts are not supported".to_string(),
+                    connector: "stripe",
+                }
+                .into()),
+            },
+            _ => Err(errors::ConnectorError::NotImplemented(
+                "This payment method is not implemented for Nomupay".to_string(),
+            )
+            .into()),
         }
     }
 }
 
-//TODO: Fill the struct with respective fields
-#[derive(Default, Debug, Clone, Serialize, Deserialize)]
-pub struct RefundResponse {
-    id: String,
-    status: RefundStatus,
-}
-
-impl TryFrom<RefundsResponseRouterData<Execute, RefundResponse>> for RefundsRouterData<Execute> {
+// PoRecipientAccount response
+impl<F> TryFrom<PayoutsResponseRouterData<F, OnboardTransferMethodResponse>>
+    for PayoutsRouterData<F>
+{
     type Error = error_stack::Report<errors::ConnectorError>;
     fn try_from(
-        item: RefundsResponseRouterData<Execute, RefundResponse>,
+        item: PayoutsResponseRouterData<F, OnboardTransferMethodResponse>,
     ) -> Result<Self, Self::Error> {
         Ok(Self {
-            response: Ok(RefundsResponseData {
-                connector_refund_id: item.response.id.to_string(),
-                refund_status: enums::RefundStatus::from(item.response.status),
+            response: Ok(PayoutsResponseData {
+                status: Some(PayoutStatus::RequiresCreation),
+                // connector_payout_id: item.data.request.connector_payout_id.clone(),
+                connector_payout_id: Some(item.response.id),
+                payout_eligible: None,
+                should_add_next_step_to_process_tracker: false,
+                error_code: None,
+                error_message: None,
             }),
             ..item.data
         })
     }
 }
 
-impl TryFrom<RefundsResponseRouterData<RSync, RefundResponse>> for RefundsRouterData<RSync> {
+// PoFulfill Request
+impl<F> TryFrom<(&PayoutsRouterData<F>, FloatMajorUnit)> for NomupayPaymentRequest {
     type Error = error_stack::Report<errors::ConnectorError>;
     fn try_from(
-        item: RefundsResponseRouterData<RSync, RefundResponse>,
+        (item, amount): (&PayoutsRouterData<F>, FloatMajorUnit),
     ) -> Result<Self, Self::Error> {
-        Ok(Self {
-            response: Ok(RefundsResponseData {
-                connector_refund_id: item.response.id.to_string(),
-                refund_status: enums::RefundStatus::from(item.response.status),
+        let source_id = match item.connector_auth_type.to_owned() {
```

Comment:
We can use TryFrom implementation for auth_type

Reply:
done